fedimintd/src/lib.rs

```diff
@@ -453,9 +453,12 @@ pub fn default_modules(
             local: EmptyGenParams::default(),
             consensus: MintGenParamsConsensus::new(
                 2,
-                // TODO: wait for clients to support the relative fees and set them to
-                // non-zero in 0.6
-                fedimint_mint_common::config::FeeConsensus::zero(),
+                if is_env_var_set(FM_ENABLE_MINT_BASE_FEES_ENV) {
+                    fedimint_mint_common::config::FeeConsensus::new(0)
+                        .expect("Relative fee is within range")
+                } else {
+                    fedimint_mint_common::config::FeeConsensus::zero()
```

Comment:
`new(0)` and `zero()` being different constructors is quite confusing. Maybe there's a better name than `new`?

Reply:
Would make it a constant instead of a function. `impl Foo { pub const ZERO : Self = Self(0); /* .... */` or something like it.